pyqtorch/utils.py

```diff
@@ -741,3 +743,15 @@ class SolverType(StrEnum):
 
     KRYLOV_SE = "krylov_se"
     """Uses Krylov Schrodinger equation solver"""
+
+
+def is_parametric(operation: pyq.Sequence) -> bool:
```

Comment:
I would import these types from PyQ, just for the sake of readability.

Reply:
yes, I tried that but the circular import error didn't allow that. I simply didn't want to fight that here

Reply:
hm OK. Thanks.